README.md

```diff
@@ -30,6 +30,32 @@ make demo
 The build script will then download data file for Doom automatically. SDL2 based window
 should appear when Doom is loaded and executed.
 
+## riscv-arch-test
+
+The RISC-V Architectural Tests, [riscv-arch-test](https://github.com/riscv-non-isa/riscv-arch-test) the basic set of tests that can ensure the riscv model's behaviour that matches RISC-V specifications while implementing specific software.(not a **substitute for rigorous design verification**)
```

Comment:
Add comma (`,`) after `riscv-arch-test`. Also, should add "is" before "the basic set of .."